frontend/views/containers/proposals/Mincome.vue

```diff
@@ -40,7 +44,9 @@ export default {
   data () {
     return {
       form: {
-        incomeProvided: null
+        incomeProvided: null,
+        error: false,
+        response: ''
```

Comment:
[Style rule](https://github.com/okTurtles/group-income-simple/blob/master/docs/Style-Guide.md#how-to-organize-data-): only vuelidate-validated data goes into `form` key, the rest should go under `ephemeral`

Reply:
I followed how we have in other files (ex: login and signup)... I fixed there too.